quickfeed.code-workspace

```diff
@@ -53,6 +53,9 @@
 		"[javascript]": {
 			"editor.defaultFormatter": "vscode.typescript-language-features"
 		},
+		"[scss]": {
```

Comment:
We migrated away from workspace file and instead use the `vscode/settings.json` instead now. You should revert this change and merge master into your PR before adding the formatter to `settings.json` instead.